__tests__/components/NetworkSwitcher.test.js

```diff
@@ -31,21 +31,21 @@ describe('NetworkSwitch', () => {
   test('correctly renders MainNet initially', () => {
     const { wrapper } = setup()
 
-    const networkSelectorElement = wrapper.find('select').getElement()
+    const networkSelectorElement = wrapper.find('button[data-value="MainNet"]')
```

Comment:
I'm not sure this is still testing is MainNet is selected on init

Reply:
The new structure of the dropdown changed the nature of this test. I've changed it to target the button that represents MainNet. The test will fail if the MainNet button does not have exactly 3 children, which means it does not have the selected div in it.